TaskAdressProject/src/AdressPJ.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class AdressPJ {
```

Comment:
What is PJ?

Reply:
This is the name of the project on PC and I mistakenly used it here :)
I already changed it